.copr/Makefile

```diff
@@ -19,6 +19,12 @@ installdeps:
 
 
 srpm: installdeps
```

Comment:
I think you missed to delete above line `srpm: installdeps`

Reply:
Right! Thanks Sandro.

Reply:
fixed